packages/eslint-config-varnish/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
please rename the folder to packages/varnish-eslint-config

Reply:
yep on it, sorry i was pushing edits in batches so the commit suggestions and local branch didn't diverge much! let me do a final rename push

Reply:
no problem!

Reply:
done! hoping i caught all of them :D

Reply:
looking :-P